cmd/src/main.go

```diff
@@ -21,7 +29,21 @@ Usage:
 The options are:
 
 	-config=$HOME/src-config.json    specifies a file containing {"accessToken": "<secret>", "endpoint": "https://sourcegraph.com"}
+      You can use "${}" syntax to reference environment variables, even on Windows,
+      but you may wish to use single-quotes to protect that syntax from shell expansion:
```

Comment:
Huh, so you are just documenting behavior that already existed here, right?

I am a little bit confused why anyone would want to do this since by default the shell would already do this expansion, right? It is also a bit confusing that this only happens for the config variable.

Not something for you to worry about, but for @slimsag to follow up on.

Reply:
That's right, and I did so because I would be gravely surprised by that behavior. The alternative would be to remove the [`os.ExpandEnv`](https://github.com/sourcegraph/src-cli/blob/3.4.0/cmd/src/main.go#L75) which is the source of that behavior and hope [no one's workflow depended on it](https://xkcd.com/1172/)

The counterbalance to "why would anyone want that" is that it _may_ harmonize some Windows behavior, since the env-expansion story is all over the place in that setup (`-config=%FOO%` if cmd, and then presumably `-config="${env:FOO}..."` if running under PS), but even under PS one would need to single-quote to avoid the dollar-brace from being consumed by PS itself

Reply:
Yes, the reason we `os.ExpandEnv` this string is because our docs imply that e.g. `-config=$HOME/src-config.json` will work, among many other situations where we suggest env vars and these won't work on Windows. The other option is to start providing windows-specific help docs, but that seems like a larger change that is unlikely to be maintained going forward. Our windows support is iffy, certainly, this just makes it an ounce nicer.

However, I do not think this needs to be documented behavior, honestly. I doubt anyone is going to be trying to pass a literal `$FOO` as part of the config file path. This feels like documenting something that nobody is ever going to run into.

Suggestion: Please remove this doc addition here OR add it to an extended help flag.